src/components/AuthModals/ResetPasswordModal/index.js

```diff
@@ -34,15 +39,43 @@ const ResetPasswordModal = (props) => {
         [inputData]
     );
 
-    if (!isVisible) {
-        return null;
-    }
+    const handlePasswordReset = useCallback(async () => {
+        try {
+            const result = await resetPassword({
+                username: email,
+                identifier,
+                password: inputData.newPassword,
+                rePassword: inputData.verifyNewPassword,
+            });
+            if (result) {
+                closeModal();
+                Toast.show('Password Reset Successfull!', Toast.SUCCESS);
+            }
+        } catch (err) {
+            console.log(err);
+            Toast.show(
+                err && err.error
+                    ? err.error
+                    : err && err.errors
+                        ? err.errors[0]
+                        : 'Invalid Password !',
```

Comment:
You can use `getErrorMessage` function from react-arsenal's `utils/error.js`

Reply:
No conditions for error.errors in utils/error.js